File: Glide/app/src/main/java/com/example/sample1/MainActivity.java
package com.example.sample1;

import androidx.appcompat.app.AppCompatActivity;

import android.net.Uri;
import android.os.Bundle;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.bumptech.glide.load.model.GlideUrl;
import com.bumptech.glide.load.model.LazyHeaders;

import javax.crypto.spec.IvParameterSpec;

public class MainActivity extends AppCompatActivity {
    String url;
    ImageView imageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        imageView = findViewById(R.id.image);
        url = "https://via.placeholder.com/600/92c9512";
        GlideUrl url = new GlideUrl("https://via.placeholder.com/600/92c9512", new LazyHeaders.Builder()
                .addHeader("User-Agent", "your-user-agent")
                .build());

        Glide.with(this).load(url).into(imageView);

    }
}

File: KotlinDaggerHilt/app/src/main/java/com/example/kotlin_dagger_hilt/presentation/MainActivity.kt
package com.example.kotlin_dagger_hilt.presentation

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.kotlin_dagger_hilt.R
import com.example.kotlin_dagger_hilt.domain.GetRetrofitData
import javax.inject.Inject


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }
}